keyboards/crkbd/keymaps/cameronjlarsen/features/oneshot.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
```

Comment:
Could you add a GPL2+ compatible license header here?  All of the source files need to have a license explicitly listed.

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Reply:
GPL2 license header has been added